pkg/github/prcreation/prcreation.go

```diff
@@ -121,7 +121,7 @@ func (o *PRCreationOptions) UpsertPR(localSourceDir, org, repo, branch, prTitle,
 		repo,
 		prTitle,
 		prBody,
-		prTitle,
+		remoteBranch,
```

Comment:
What is this `remoteBranch` and why is it used as value for `matchTitle`?

Reply:
I added the remoteBranch as I couldn't reuse the prTitle as a branch which was being done previously.(I think it was because the prTitle was too long)
In my case the remoteBranch is the prefix for my prTitle and would match the title without the timestamp. But I see how this would be incorrect and I'll change the name from remoteBranch to matchTitle

Reply:
okay, would you then mind to
* Rename the variable to something that reflects its purpose (`matchTitle`)
* Use the [functional opts pattern](https://github.com/tmrts/go-patterns/blob/master/idiom/functional-options.md) to make this an optional option